### PR TITLE
[WIP] Creating good readme

### PR DESCRIPTION
**Pull request by Denmey**
See issue #1 

### Tasks
- [ ] Come up with project's name
- [x] Write an project description
- [x] Add dependencies
- [x] Create a link to contributing.md
- [ ] Specification?

### CODE REVIEW

**Comment by makometr** [comment]
Is contributing.md really needed?

**Comment by Denmey** [comment]
Can we delete this cool  Team-members part and merge it?

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
\ No newline at end of file
+# Tetris
+Project "Tetris" is a classic game with GUI written in **C++** with **SFML** graphics library.
+
+## Compiling
+
+### Dependencies
+| Dependency | Version |
+|------------|---------|
+| GCC        | 7.3+    |
```
Comment (Transliteration) [inline README.md:9]: Хаха, у меня gcc 6.3, и тааак сойдёт

Comment (makometr) [reply README.md:9]: У меня тоже 6.3

**Review by Transliteration — APPROVED** [review]
Nice man

**Review by makometr — APPROVED** [review]
So good readme for the empty project:)
Well, i agree

**Pull request merged**